server/lib/endpoints/graphql/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const OAuthEndpoint = require('../oauth-endpoint');
+
+/**
+ * @see https://github.com/DoSomething/graphql
+ */
+class GraphQLEndpoint extends OAuthEndpoint {
+  constructor(client) {
+    super(client);
+    this.url = `${this.client.config.services.graphql.baseUri}/graphql`;
+  }
+  /**
+   * @param  {Object} query
+   * @return {Promise}
+   */
+  get(query) {
```

Comment:
I think we might find that a REST client isn't the most useful wrapper for GraphQL requests, since you'd have to handle encoding queries by hand. We use [Apollo Client](https://www.apollographql.com/docs/react/) in Phoenix, which handles caching & persisted queries out of the box. Something like [graphql-request](https://github.com/prisma/graphql-request) may be a good place to start.

Reply:
I've removed this.. was thinking that it'd be useful for Gambit Content to query a  `gatewayClient.GraphQL` instead of a `gatewayClient.Rogue.Campaigns` -- but also definitely see how the encoding you mean is not fun when using it over in https://github.com/DoSomething/gambit-admin/pull/86/commits/ab1076018d2ca0291f445f66727251b2255ad465